package.json

```diff
@@ -1,19 +1,18 @@
 {
   "name": "slate-edit-code",
   "description": "A Slate plugin to handle code blocks editing.",
-  "version": "0.10.2",
+  "version": "0.11.0",
   "license": "Apache-2.0",
   "repository": "git://github.com/GitbookIO/slate-edit-code.git",
   "main": "./dist/index.js",
   "dependencies": {
     "detect-indent": "^4.0.0",
     "detect-newline": "^2.1.0",
     "ends-with": "^0.2.0",
-    "immutable": "^3.8.1"
-  },
-  "peerDependencies": {
-    "slate": "0.x.x"
+    "immutable": "^3.8.1",
+    "slate": "^0.22.10"
```

Comment:
@SamyPesse ?

Reply:
I prefer that we keep `slate` as a peerDependencies. Since this module is a plugin for slate, it helps avoid duplication of `slate` and let the parent project controls the `slate` dependency.